### PR TITLE
Add Taiko Suiji Cup 2020 article

### PR DESCRIPTION
**Pull request by fajar13k**
- Add TSC 2020 article
- Minor changes inside Taiko Suiji Cup index

### CODE REVIEW

File: wiki/Tournaments/TSC/2020/en.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+| Streamer | ![][flag_BR] [AnonX32](https://osu.ppy.sh/users/2730270), ![][flag_US] [ITotallyGetThat](https://osu.ppy.sh/users/8770622), ![][flag_NL] [oliebol](https://osu.ppy.sh/users/2756335), ![][flag_DE] [Sayira](https://osu.ppy.sh/users/7253958) |
+| Designer | ![][flag_KR] [POCARI SWEAT](https://osu.ppy.sh/users/5082685) |
+| Statistician | ![][flag_PL] [Acrith](https://osu.ppy.sh/users/389880) |
+| Wiki editor | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
```
Comment (TPGPL) [inline wiki/Tournaments/TSC/2020/en.md]: capitalise both words in position column

File: wiki/Tournaments/TSC/2020/en.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+*The participants will be revealed after the registration phase ends.*
+
+## Mappools
+
+*The mappools will be revealed one week before each particular stage.*
+
+## Ruleset
+
+### General rules
```
Comment (TPGPL) [inline wiki/Tournaments/TSC/2020/en.md]:
```suggestion
### General Rules
```

File: wiki/Tournaments/TSC/2020/en.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    - Mappool B will contain 2 Challenge Maps, 1 DoubleTime, 1 Easy Mod, 1 Hidden+HardRock, and the Tiebreaker.
+    - Any maps from Mappool B **is not bannable**.
+    - Challenge bracket is map that contains unusual pattern, unconventional timing, high bpm speed, and will be played under NoMod.
+11. The Qualifiers Mappool however will have only one pool instead of two. It will contain 2 NoMod maps. 1 Hidden map, 1 HardRock map, and 1 Challenge map. Maps will be all played in the qualifier lobby, and **are not bannable**.
+12. One player from each seed **must play** at the same time in the 3v3 setting. This means (ABC) vs. (ABC). NOT (AAB) vs. (ABB)
+13. **C's may play in place of B's, and B's in place of A's** only in situations where the players cannot be active (i.e. emergencies, vacation, etc).
+14. All match times are in **UTC**.
+15. No staff members are allowed to play other than streamer, commentator, and graphic designer.
+
+### Match Regulations
```
Comment (TPGPL) [inline wiki/Tournaments/TSC/2020/en.md]:
```suggestion
### Match regulations
```

File: wiki/Tournaments/TSC/2020/en.md (hunk 1 above)
Comment (cl8n) [reply wiki/Tournaments/TSC/2020/en.md]: dont do that

**Pull request merged**